contrib/avro/src/python/pants/contrib/avro/targets/java_avro_library.py

```diff
@@ -0,0 +1,21 @@
+# coding=utf-8
+# Copyright 2017 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+from pants.backend.jvm.targets.exportable_jvm_library import ExportableJvmLibrary
+from pants.base.payload import Payload
+
+
+class JavaAvroLibrary(ExportableJvmLibrary):
```

Comment:
Is ExportableJvmLibrary the right subclass? I think that the exportable target would be the synthetic JavaLibrary.

I am a bit behind on the recent reworking of exports, though. But FWICT, publishing a java_library that depends on an AvroLibrary would fail because the publish class would think the `provides` was defined 2x.

Reply:
I don't recall why I chose that. I've switched it to `JvmTarget` just like `JavaThriftLibrary`.